toc-ja.md

```diff
@@ -2,25 +2,25 @@
 
 File | Description
 ---|---
-[./snippets/add-element-to-iframe.js](./snippets/add-element-to-iframe.js) |
-[./snippets/add-text-to-iframe.js](./snippets/add-text-to-iframe.js) |
+[./snippets/add-element-to-iframe.js](./snippets/add-element-to-iframe.js) | iframeに要素を追加したいときにご利用ください。
```

Comment:
I also thought it would be a good idea to describe in what cases you should add elements to the iframe. (For example, `何らかの理由により、Autify のテスト実行で値の入力ができないときにご利用ください。` is good!)

Reply:
Thanks!  However, I've never had a situation where I needed this and I wondered how I should put it - TSEs were also wondering what were the specific use cases for these iframe snippets 😅

Reply:
IMO we can just remove those snippets. It seems useless